### ISSUE TITLE
🐜 Accessibility Bug - Random URL read outs on error message

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/gc-digital-talent#5479]
## ❌ Current behaviour (issue)
Error messages contain long URLs in the text read out of the messages (e.g., when someone enters an incorrect telephone number sequence) by JAWS.

## 🗺️ Location 

Profile form section (e.g. About Me)

## 💻 AT Configuration

JAWS, Windows, Edge 

## 📋 How to reproduce

1.	Open JAWS
2.	Navigate using your keyboard to profile and then About Me Section 
3.	Enter an incorrect format in the telephone number or field 
4.	Tap the Save and go back.
5.	Listen to the error message presented for the extra url read out. 

## 📹 Fable Video Clip 
[Extended Error Messages ](https://app.makeitfable.com/clip/63c19e73a15f520013ac0bba)

## ✔ Expected behaviour (fix)
Error message texts are read without url details. 

## ✅ Acceptance Criteria
- [ ] Error message texts are read without url details

**Comment by petertgiles** [comment]
Hmm, I watched the video but I'm not sure what the solution is.  As far as I know we really want to be able to click on the name of the errored field to go right there, which is what the URL is for.  What is the proposed solution?

**Comment by RM-1978** [comment]
The Fable tester suggested removing the readout from the tag.

**Comment by petertgiles** [comment]
Sorry if this is obvious. :sweat_smile: Do you have a resource on how to remove the readout from the tag that we can attach to this issue?
![image](https://user-images.githubusercontent.com/8978655/228536236-9f6e162b-f11b-4ae7-9019-d949fc25b554.png)

**Comment by RM-1978** [comment]
Hi @petertgiles, here are the resources that I found, but a note that @substrae might have more thoughts given his work in Hydrogen: 
 - [Using the CSS ](https://a11y-guidelines.orange.com/en/articles/accessible-hiding/#:~:text=The%20conventional%20way%20is%20to,by%20Assistive%20technologies%20(AT).)
 - [Disable HTML elements](https://stackoverflow.com/questions/39256144/aria-for-adding-or-removing-a-div) 

warmly, RM

**Comment by RosalieM1978** [comment]
@esizer, can you reproduce the error to see if this issue can still be found on the Profile information page?

**Comment by esizer** [comment]
> @esizer, can you reproduce the error to confirm if this issue can still be found on the Profile information page?

Unable to reproduce 👎
